apps/speed.c

```diff
@@ -1565,8 +1570,10 @@ int speed_main(int argc, char **argv)
         for (i = 0; i < ALGOR_NUM; i++)
             if (i != D_EVP)
                 doit[i] = 1;
+#ifndef OPENSSL_NO_RSA
```

Comment:
As an aside no-rsa isn't a recognised option in 1.1.0. We've left in the existing OPENSSL_NO_RSA guards for now in case we want to resurrect it - but as it stands no-rsa would not build if we enabled it. No action necessary for this comment.